docs/deployment/helm.md

```diff
@@ -86,14 +86,7 @@ We have introduced the following Chart parameters.
 
 ### General parameters
 
-| Name | Type | Default | description |
-| ---- | ---- | ------- | ----------- |
-| `image.repository` | string | `{{ site.container_image | split: ":" | first }}` | NFD image repository |
-| `image.tag` | string | `{{ site.release }}` | NFD image tag |
-| `image.pullPolicy` | string | `Always` | Image pull policy |
-| `imagePullSecrets` | list | [] | ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this PodSpec. If specified, these secrets will be passed to individual puller implementations for them to use. For example, in the case of docker, only DockerConfig type secrets are honored. [More info](https://kubernetes.io/docs/concepts/containers/images#specifying-imagepullsecrets-on-a-pod) |
-| `nameOverride` | string |  | Override the name of the chart |
-| `fullnameOverride` | string |  | Override a default fully qualified app name |
+Check [nfd-chart-params][nfd-chart-params] for a fully detailed list.
```

Comment:
I think we should still have a table of Helm parameters for the _operator_ itself